server/channels/api4/ldap.go

```diff
@@ -90,6 +92,31 @@ func testLdap(c *Context, w http.ResponseWriter, r *http.Request) {
 	ReturnStatusOK(w)
 }
 
+func testLdapConnection(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
nit: I think it should go all uppercase (i.e. LDAP) in idiomatic Go casing but I guess we haven't been very consistent so far.